<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddBookDialog</class>
 <widget class="QDialog" name="AddBookDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавить книгу</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLineEdit" name="titleEdit">
     <property name="placeholderText">
      <string>Название книги</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="authorEdit">
     <property name="placeholderText">
      <string>Автор</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Год издания:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="yearSpin">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Страниц:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="pagesSpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="placeholderText">
      <string>Описание книги</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Экземпляров:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="copiesSpin">
       <property name="minimum">
        <number>1</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="conditionEdit">
     <property name="placeholderText">
      <string>Состояние (например, "Новая")</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Доступно:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="availableSpin">
       <property name="minimum">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
